README.md

```diff
@@ -1,121 +1,3 @@
-<br /><a name="f21">21</a>: State transitions are calculated to be (200*73*10)*500000/576 bytes, per block, increasing blocksize by 807291 bytes each block, 17% of the total blocksize.
-<br /><a name="f22">22</a>: Tier-2 conceptually is able to handle much larger block propagation, due to having much higher quality network connections. We plan on having large farms of co-located servers, eventually connected using ethernet hand-off. Masternodes located in the same facility can be used for local syncing, allowing more rapid propagation of blocks to catch new servers up. Tier-1 remains permissionless, but also becomes vastly more expensive. Processing of blocks uses custom hardware, allowing for us to process transaction/transition signatures in a parallelized way.
-<br /><a name="f23">23</a>: 868 transactions per block, if assumed 1 transaction per day, per user (lots of assumptions // we need more data). (5TX/SEC + 2TS/SEC)
-<br /><a name="f24">24</a>: Second-generation Evolution wallets are software based wallets which expand the object schema to allow internal wallet-to-wallet custom relationships for custom applications such as allowing businesses to have custom wallets for their application. This will allow streamlined applications with easy to use features within subsets of the whole dash economy.
-<br /><a name="f25">25</a>: DashDrive object writes require fees for each object written to the system. The system is designed to use an initial burn of coins, which create a small credit that is tracked in the system, debits are introduced for each operation that is done in relation to Evolution.
-<br /><a name="f26">26</a>: Using specialized state transition for maintenance functions Each component of masternode will run redundantly under a load balancer to allow for individual component failure without causing a cascade which creates an outage
-<br /><a name="f27">27</a>: Using third party API keys, allowing user to setup keys in settings then use from wallet seamlessly without ever leaving Dash.
-<br /><a name="f28">28</a>: By leveraging dapi and third party tools, we can deploy addons and a marketplace, which allows custom third party services to integrate directly in the wallet such as debit/credit cards and fiat bank account support
-<br /><a name="f29">29</a>: High-end quality processor like the Xeon, multi-core. Recommend to at least have 8 cores.
-<br /><a name="f30">30</a>: Sporks, our software based forking mechanism will be controlled by a few members of the core-team. Three of five keys will be used as a threshold for activation of specific core-daemon features. This is to stop accidents happening, damaging the network due to neglect or misunderstanding and outright network abuse.
+The Official Dash Roadmap can be found at https://www.dash.org/roadmap/. Previous versions can be found in the history of this Git repository.
```

Comment:
> the history of this Git repository

Let's maybe provide a direct link for non-technical people?
https://github.com/dashpay/dash-roadmap/tree/eb7f6bf4c0700fd2568a22b9a0b97881c55fe4da

Reply:
Yeah, I agree. Or a link to the commits.